.github/workflows/refresh-cache.yml

```diff
@@ -0,0 +1,95 @@
+
```

Comment:
Why do we refresh our cache? To obtain new versions of packages? Because there are seperate tools for that one I think

Reply:
Various reasons. Obtained new versions of packages, integrate new packages into the cache, make sure all the tests are running from the same cache. Because mamba solving is not stable, it's important to keep all the tests in sync by periodically refreshing the cache that they all use. Besides, the cache limit is at 10GB which we bump up against about weekly, instead of trying to do very clever cache invalidation I thought it was much more maintainable to just clean out and regenerate the cache every week, that way we make sure we keep within our limits and that everything uses the up to date versions of everything.

Reply:
It seems that dependabot does not do mamba / conda, so I think this is way better than nothing :)